tracing-tower-http/examples/tower-h2-server.rs

```diff
@@ -40,11 +40,7 @@ impl Body for RspBody {
     type Data = <Bytes as IntoBuf>::Buf;
     type Error = h2::Error;
 
-    fn is_end_stream(&self) -> bool {
```

Comment:
Why was this removed?

Reply:
the trait changed upstream; the old code no longer complied and I wanted a green CI build.